src/app/search/search.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="main">
+  <section>
+    <header>
+      <h2>Do a search for...</h2>
+
+      <div class="form-group">
+        <input (change)="search(1)" [(ngModel)]="searchTerm" placeholder="search by title or description"/><!-- TODO: delay search until finish typing -->
+      </div>
+
+      <div class="form-group">
+        <span class="left">
+          <label [attr.for]="searchSeries">Filter by Series</label>
+          <select id="searchSeries" [(ngModel)]="searchSeriesId" (ngModelChange)="searchBySeries()">
+            <option *ngFor="let seriesId of allSeriesIds" [value]="seriesId">{{allSeries[seriesId] ? allSeries[seriesId].title : 'No Series'}}</option>
+          </select>
+          <!-- TODO: there should be a way to clear this -->
```

Comment:
If that `angular2-select` works, I'd say go for it.  There are a couple other competing select2 ports ... `ng2-select2`, `angular2-select2`.

The most popular is probably `ng2-select` by valor-software ... but it looks like it may not be compatible with the 2.0.0-final release yet.  (I see an issue for it, though).